django/contrib/postgres/forms/jsonb.py

```diff
@@ -11,6 +11,10 @@ class InvalidJSONInput(six.text_type):
     pass
 
 
+class JsonStr(six.text_type):
```

Comment:
_Minor:_ `JsonStr` could be renamed to `JSONStr` or `JSONString` to be consistent with the naming of `InvalidJSONInput` class above.